shenyu-client/shenyu-client-http/shenyu-client-springcloud/src/test/java/org/apache/shenyu/client/springcloud/init/SpringCloudClientBeanPostProcessorTest.java

```diff
@@ -58,29 +58,31 @@ public final class SpringCloudClientBeanPostProcessorTest {
 
     private final MockedStatic<RegisterUtils> registerUtilsMockedStatic = mockStatic(RegisterUtils.class);
 
-    private final SpringCloudClientTestBean springCloudClientTestBean = new SpringCloudClientTestBean();
+    @Mock
+    private ApplicationContext applicationContext;
+
+    private ContextRefreshedEvent contextRefreshedEvent;
 
     @BeforeEach
     public void init() {
         when(env.getProperty("spring.application.name")).thenReturn("spring-cloud-test");
+        contextRefreshedEvent = new ContextRefreshedEvent(applicationContext);
     }
 
     @Test
     public void testShenyuBeanProcess() {
         registerUtilsMockedStatic.when(() -> RegisterUtils.doLogin(any(), any(), any())).thenReturn(Optional.of("token"));
         // config with full
-        SpringCloudClientBeanPostProcessor springCloudClientBeanPostProcessor = buildSpringCloudClientBeanPostProcessor(true);
-        assertThat(springCloudClientTestBean, equalTo(springCloudClientBeanPostProcessor.postProcessAfterInitialization(springCloudClientTestBean, "springCloudClientTestBean")));
+        SpringCloudClientBeanListener springCloudClientBeanListener = buildSpringCloudClientBeanPostProcessor(true);
+        springCloudClientBeanListener.onApplicationEvent(contextRefreshedEvent);
         registerUtilsMockedStatic.close();
```

Comment:
missed assert judge.

Reply:
Method has no return value. I added the call count vetify.